Core/Utilities/RabbitMQAdmin.py

```
@@ -0,0 +1,138 @@
+  users = ret['Value']
+  users = users.split('\n')
+  # the rabbitMQ user list is given in the format:
+  # user_name [usr_tag]
+  # I remove [usr_tag] part.
+  # Also only non-empty users are proceeded further.
+  # Empty users can appear, cause every new line was
+  # treated as a new user.
+  users = [ re.sub('\\t\[\w*\]$','',u) for u in users if u]
+  return S_OK(users)
+
+def setUserPermission(user):
+  return executeRabbitmqctl('set_permissions','-p','/', user, '\".*\"','\".*\"','\".*\"')
+
+def clearUserPassword(user):
+  return executeRabbitmqctl('clear_password', user)
+
+def setUsersPermissions(users):
+  successful = {}
+  failed = {}
+  for u in users:
+    ret = setUserPermission(u)
+    if ret['OK']:
+      successful[u] = ret['Value']
+    else:
+      print "Problem with permissions:%s" % ret['Message']
+      failed[u] = "Permission not set"
+  return S_OK({'Successful': successful, 'Failed': failed})
+
+def addUsersWithoutPasswords(users):
+  successful = {}
+  failed = {}
+  for u in users:
+    ret = addUserWithoutPassword(u)
+    if ret['OK']:
+      successful[u] = ret['Value']
+    else:
+      print "Problem with adding user:%s" % ret['Message']
+      failed[u] = "User not added"
+  return S_OK({'Successful': successful, 'Failed': failed})
+
+def addUsers(users):
+  """Adds users to the RabbitMQ internal database.
+  """
+  successful = {}
+  failed = {}
+  for u in users:
+    ret = addUser(u)
+    if ret['OK']:
+      successful[u] = ret['Value']
+    else:
+      print "Problem with adding user:%s" % ret['Message']
+      failed[u] = "User not added"
+  return S_OK({'Successful': successful, 'Failed': failed})
+
+def deleteUsers(users):
+  """Deletes users from the RabbitMQ internal database.
+  """
+  successful = {}
+  failed = {}
+  for u in users:
+    ret = deleteUser(u)
+    if ret['OK']:
+      successful[u] = ret['Value']
+    else:
+      print "Problem with adding user:%s" % ret['Message']
+      failed[u] = "User not added"
```

Don't you need to propagate the actual error ?